corpdir-srv-lde/src/main/java/com/github/bordertech/corpdir/srv/lde/dummy/api/ContactReadOnlyServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package com.github.bordertech.corpdir.srv.lde.dummy.api;
```

Comment:
These `dummy` implementations could be in their own module and it might be better to refer to them as `mock`. Being in their own module would also make it easier to use the mock implementation in other situations (eg junits).

Reply:
Ok, Moved dummy api impl into seperate module